docs/source/basics/customizing_interface.md

```diff
@@ -425,32 +120,84 @@ The custom layout is specified using the `flow` parameter of the {class}`~latch.
 
 - {class}`~latch.types.metadata.Fork` shows a set of mutually-exclusive alternatives. The alternatives are specified as a list of {class}`~latch.types.metadata.ForkBranch`, each of which displays a child flow when active and nothing otherwise. Each branch is identified by a unique key. This key is passed to the workflow is a `str`-typed parameter so the user selection can be used to change runtime behavior
 
----
+<details>
+<summary>Flow Example</summary>
 
-## Adding your workflow to a biological domain on Latch
+![Flow Example 1](../assets/flow-example/flow_example_1.png)
 
-For public workflows, you may want to classify your workflow to a biological domains to make it easier for future users to discover.
+![Spoiler Example](../assets/flow-example/flow_example_spoiler.png)
 
-To do so, you can use the `tags` property of `LatchMetadata`.
+The `LatchMetadata` can be defined as follows:
 
 ```python
+from latch.types import LatchMetadata, LatchParameter
+from latch import workflow
+
 metadata = LatchMetadata(
-    ...
-    tags=["NGS", "MAG"],
-    ...
+    display_name="fastp - Flow Tutorial",
+    author=LatchAuthor(
+        name="LatchBio",
+    ),
+    parameters={
+        "sample_fork": LatchParameter(),
+        "paired_end": LatchParameter(
+            display_name="Paired-end reads",
+            description="FASTQ files",
+            batch_table_column=True,
+        ),
+        "single_end": LatchParameter(
+            display_name="Single-end reads",
+            description="FASTQ files",
+            batch_table_column=True,
+        ),
+        "output_directory": LatchParameter(
+            display_name="Output directory",
+        ),
+    },
+    flow=FLOW,
 )
+
+@workflow(metadata)
+def fastp(
+    sample_fork: str,
+    paired_end: PairedEnd,
+    single_end: Optional[SingleEnd] = None,
+    output_directory: str = "fastp_results",
+) -> LatchDir:
+    ...
 ```
 
-Below is a list of commonly used domains on Latch. For best practices, you should tag your workflow with an existing domain instead of creating a new one.
+where `FLOW` is a list of flow elements:
 
-- Aggregator
-- COVID
-- CRISPR
-- Epigenetics
-- Guide Design
-- Library Screen
-- MAG
-- NGS
-- Nextflow
+```python
+FLOW = [
```

Comment:
just show this inline? it's kind of weird to split things up like this

also would rather `flow` than `FLOW` tho that's entirely stylistic